composer.json

```diff
@@ -10,13 +10,13 @@
         }
     ],
     "require": {
-        "zendframework/zend-http": "~2.2.5",
-        "zendframework/zend-stdlib": "~2.2.5",
-        "zendframework/zend-uri": "~2.2.5"
+        "php": ">=5.5.0",
+        "zendframework/zend-stdlib": "~2.2.5"
     },
     "require-dev": {
         "phpunit/PHPUnit": "3.7.*",
         "satooshi/php-coveralls": ">=0.6.0",
+        "psr/http-message": "1.0.*@dev",
```

Comment:
But what Dasprid is trying to tell is that Dash WILL depend on a library that implement psr-7 (Zend/Http from zf3, guzzle/http... Or whatever is available).

The dependency to psr7 will be implicit.

Envoyé de mon iPhone

> Le 6 avr. 2015 à 18:55, Daniel Gimenes <notifications@github.com> a écrit :
> 
> In composer.json:
> 
> >      },
> >      "require-dev": {
> >          "phpunit/PHPUnit": "3.7.*",
> >          "satooshi/php-coveralls": ">=0.6.0",
> > +        "psr/http-message": "1.0.*@dev",
> Since you're typehinting against psr/http-message interfaces, your package depends on psr/http-message. You should not rely on others to install your own dependencies.
> 
> —
> Reply to this email directly or view it on GitHub.
>